migrations/versions/90f1af83d9b6_add_reviewer_role.py

```diff
@@ -0,0 +1,23 @@
+"""Add reviewer role
+
+Revision ID: 90f1af83d9b6
+Revises: 5d619660cfa7
+Create Date: 2016-05-17 02:16:30.097950
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '90f1af83d9b6'
+down_revision = '5d619660cfa7'
+
+from alembic import op
+
+
+def upgrade():
+    """Add reviewer role."""
+    op.execute("INSERT INTO roles (name) VALUES ('reviewer');")
+
+
+def downgrade():
+    """Remove reviewer role."""
+    op.execute("DELETE FROM roles WHERE name='reviewer';")
```

Comment:
Do we have cascades set up? If not we'll need to remove records from `roles_users` as well.